packages/components/src/index.native.js

```diff
@@ -15,6 +15,7 @@ export { default as Dashicon } from './dashicon';
 export { default as Dropdown } from './dropdown';
 export { default as DropdownMenu } from './dropdown-menu';
 export { default as FocalPointPicker } from './focal-point-picker';
+export { default as FontSizePicker } from './font-size-picker';
```

Comment:
On Android I'm getting this error:

```javascript
 ERROR  TypeError: Cannot read property 'createSlotFill' of undefined, js engine: hermes
```
I think it's because the `FontSizePicker` import is before:

```javascript
export {
	createSlotFill,
	Slot,
	Fill,
	Provider as SlotFillProvider,
} from './slot-fill';
```